test/mongo.test.js

```diff
@@ -93,7 +93,7 @@ test('increment org total amount donated from existing value', async (t) => {
 
   await mongo.updateDonatedAmount({ organizationId: orgId1.toString(), amount: 1000 })
 
-  const updatedOrg = await mongo.db.collection('organizations').findOne({ _id: orgId1 })
+  const updatedOrg = await mongo.getOrg({ organizationId: orgId1.toString() })
```

Comment:
Updated these lines of the tests to use the `getOrg` method -- this is why we didn't catch it. The tests pulled the whole org from in-memory mongo, but real-life pulled a selection of the org.